ts/components/ui/RNavScreenWithLargeHeader.tsx

```diff
@@ -88,7 +89,17 @@ export const RNavScreenWithLargeHeader = ({
     contextualHelpMarkdown,
     faqCategories,
     ...headerActionsProp
-  });
+  };
+
+  const headerProps: ComponentProps<typeof HeaderSecondLevel> = useHeaderProps(
+    canGoback
+      ? {
+          backAccessibilityLabel: I18n.t("global.buttons.back"),
+          goBack: goBack ?? navigation.goBack,
+          ...headerPropsWithoutGoBack
```

Comment:
Usually the **spread** operator is used **at the beginning of the object**, because what goes after sometimes **overrides** some properties contained in the object. This is not the case, but I suggest to always use the spread operator as the first part of the object as a best practice. What do you think?

Reply:
Done! ✅ Thanks for the suggestion!